docs/source/flow-project.rst

```diff
@@ -214,6 +214,14 @@ As shown before, all *eligible* operations can then be executed with:
 
     ~/my_project $ python project.py run
 
+The status determination is by default parallelized with threads, however this can be turned off or switched to using processes by setting a value for the ``flow.status_parallelization`` configuration key to either ``none``, ``processes`` with ``threads`` being the implicit default value.
```

Comment:
```suggestion
The status determination is by default parallelized with threads, however this can be turned off or switched to using processes by setting a value for the ``flow.status_parallelization`` configuration key.
Possible value are ``thread``, ``process`` or ``none`` with the first one being the implicit default value and the last one turning off all parallelization.
```
Further suggested improvement of the phrasing. I also noticed that `threads` and `processes` in my suggested change was in fact wrong, because signac-flow expects the singular. This is a reminder to please be very diligent when accepting suggestions. I did not double check those values when I provided the suggestion, but relied on you knowing what the correct values would be.

Reply:
Oh yes, I am sorry about that. I just got careless over there. I'll take care of it from next time.